spec/lib/daemons/rails/monitoring_spec.rb

```diff
@@ -6,36 +6,28 @@
   [Daemons::Rails::Monitoring, Daemons::Rails::Monitoring.new(Rails.root.join('lib', 'daemons'))].each do |subject|
     it "should get list of controllers" do
       controllers = subject.controllers
-      controllers.should have(1).item
+      expect(controllers.length).to eq 1
       controller = controllers[0]
-      controller.path.should == Rails.root.join('lib', 'daemons', 'test_ctl')
-      controller.app_name.should == 'test.rb'
+      expect(controller.path).to eq Rails.root.join('lib', 'daemons', 'test_ctl')
+      expect(controller.app_name).to eq 'test.rb'
     end
 
     describe "using controllers" do
-      before :each do
-        @controller = Daemons::Rails::Controller.new(Rails.root.join('lib', 'daemons', 'test_ctl'))
-        Daemons::Rails::Monitoring.any_instance.stub(:controllers).and_return([@controller])
+      it "should start controller by name" do
+        subject.start('test.rb')
```

Comment:
It should check something after start (i.e. job may create some file with timestamp which may be checked after start)

Reply:
I am planning to add more tests for the creation of log files which should address this issue.  I am not totally sure if i am creating the log files in the correct location. I used basically same code that Daemons gem has to detect the folder location for log files but sometimes i have the impression that i am creating them in a different location and the Daemons gem creates other files too in other location. Although only sometimes i had that impression. I definitely need to test more how those files are created. I basically used for that almost same code but with some minor changes.  Unfortunately i need to do this because daemons gem does not check if these files exist...it only attempts to write and If it can not write the daemon dies. Last time i verified the daemons were not dieing anymore but i am still not sure if the location of those files îs detected properly.